client/components/currency-information-for-methods/index.js

```diff
@@ -14,17 +14,19 @@ import WCPaySettingsContext from '../../settings/wcpay-settings-context';
 import InlineNotice from 'components/inline-notice';
 import PaymentMethodsMap from '../../payment-methods-map';
 
-const ListToCommaSeparatedSentencePartConverter = ( items ) => {
+const formatListOfItems = ( items ) => {
 	if ( items.length === 1 ) {
 		return items[ 0 ];
-	} else if ( items.length === 2 ) {
+	}
+
+	if ( items.length === 2 ) {
 		return items.join( ' ' + __( 'and', 'woocommerce-payments' ) + ' ' );
 	}
+
 	const lastItem = items.pop();
-	return (
-		items.join( ', ' ) +
-		__( ', and', 'woocommerce-payments' ) +
```

Comment:
Some languages might prefer _not_ to have the `,` before the last `and`. It might not be clear to translators what is being translated.

Fixing this function so that `length >= 2` is handled by the same use case.